contracts/ColonyTask.sol

```diff
@@ -310,15 +310,16 @@ contract ColonyTask is ColonyStorage, DSMath {
     Task storage task = tasks[_id];
     IColonyNetwork colonyNetworkContract = IColonyNetwork(colonyNetworkAddress);
 
+    task.finalized = true;
```

Comment:
Why has this moved?

Reply:
I made a habit of changing these types of variables before doing any logic for safety. I know it would be totally safe to do it under, but it was just for consistency. I can move it back if you want

Reply:
For reentrancy you mean? 
It's fine to leave it there I guess.